Recurly/EmptyResource.cs

```diff
@@ -0,0 +1,7 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace Recurly
+{
+    [ExcludeFromCodeCoverage]
+    public class EmptyResource : Resource { }
```

Comment:
This is necessary to facilitate setting the base type of `MakeRequest` and `MakeRequestAsync` to `Response`.

This cannot be made available to the user without introducing a breaking change.